### ISSUE TITLE
aws_kms_alias isn't imported (RPC error)

### ISSUE THREAD

**Issue by sjourdan** [GoogleCloudPlatform/terraformer#389]
### TLDR

- importing KMS key (`aws_kms_key`) works, but not the alias (`aws_kms_alias`). 
- Debug outputs a `panic: runtime error: invalid memory address or nil pointer dereference` error. 

### Environment

- terraformer: 0.8.6
- terraform: 0.12.20
- terraform-provider-aws: v2.48.0

### Details 

- `kms_key.tf` is generated properly with proper `aws_kms_key` resource.
- `aws_kms_alias` resource is not generated properly.

### Crash Output 

```
$ ./terraformer-all-darwin-amd64-0.8.6 import aws --regions=eu-west-3 --resources kms
2020/02/13 11:29:58 aws importing region eu-west-3
2020/02/13 11:29:58 aws importing... kms
2020/02/13 11:30:04 Refreshing state... aws_kms_key.tfer--a81b80d1-002D-e0ad-002D-40fa-002D-af9f-002D-60aa25b11343
[...]
2020/02/13 11:30:07 Refreshing state... aws_kms_alias.tfer--alias-002F-cs-002D-kms-002D-1
2020/02/13 11:30:07 rpc error: code = Canceled desc = context canceled
[...]
2020/02/13 11:30:07 ERROR: Unable to refresh resource tfer--alias-002F-cs-002D-kms-002D-1
2020/02/13 11:30:07 aws Connecting....
2020/02/13 11:30:07 aws save kms
2020/02/13 11:30:07 aws save tfstate for kms
```

Here's a debug snippet:

```
2020/02/13 11:42:59 Refreshing state... aws_kms_alias.tfer--alias-002F-aws-002F-dynamodb
2020-02-13T11:42:59.487+0100 [DEBUG] plugin.terraform-provider-aws_v2.48.0_x4: 2020/02/13 11:42:59 [DEBUG] Listing KMS aliases: {
2020-02-13T11:42:59.487+0100 [DEBUG] plugin.terraform-provider-aws_v2.48.0_x4:   Limit: 100
2020-02-13T11:42:59.487+0100 [DEBUG] plugin.terraform-provider-aws_v2.48.0_x4: }
2020-02-13T11:42:59.511+0100 [DEBUG] plugin.terraform-provider-aws_v2.48.0_x4: 2020/02/13 11:42:59 [DEBUG] Found KMS Alias: {
2020-02-13T11:42:59.511+0100 [DEBUG] plugin.terraform-provider-aws_v2.48.0_x4:   AliasArn: "arn:aws:kms:eu-west-3:569427490817:alias/aws/dynamodb",
2020-02-13T11:42:59.511+0100 [DEBUG] plugin.terraform-provider-aws_v2.48.0_x4:   AliasName: "alias/aws/dynamodb"
2020-02-13T11:42:59.511+0100 [DEBUG] plugin.terraform-provider-aws_v2.48.0_x4: }
2020-02-13T11:42:59.514+0100 [DEBUG] plugin.terraform-provider-aws_v2.48.0_x4: panic: runtime error: invalid memory address or nil pointer dereference
2020-02-13T11:42:59.514+0100 [DEBUG] plugin.terraform-provider-aws_v2.48.0_x4: [signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x4959dd3]
2020-02-13T11:42:59.514+0100 [DEBUG] plugin.terraform-provider-aws_v2.48.0_x4:
2020-02-13T11:42:59.514+0100 [DEBUG] plugin.terraform-provider-aws_v2.48.0_x4: goroutine 163 [running]:
2020-02-13T11:42:59.515+0100 [DEBUG] plugin.terraform-provider-aws_v2.48.0_x4: github.com/terraform-providers/terraform-provider-aws/aws.resourceAwsKmsAliasRead(0xc0002e5650, 0x5106fc0, 0xc0004c5400, 0xc0002e5650, 0x0)
2020-02-13T11:42:59.515+0100 [DEBUG] plugin.terraform-provider-aws_v2.48.0_x4:  /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-aws/aws/resource_aws_kms_alias.go:125 +0x3a3
2020-02-13T11:42:59.515+0100 [DEBUG] plugin.terraform-provider-aws_v2.48.0_x4: github.com/hashicorp/terraform-plugin-sdk/helper/schema.(*Resource).RefreshWithoutUpgrade(0xc0005aff80, 0xc000aa9c70, 0x5106fc0, 0xc0004c5400, 0xc000883710, 0xc000aa9c70, 0x0)
2020-02-13T11:42:59.515+0100 [DEBUG] plugin.terraform-provider-aws_v2.48.0_x4:  /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-aws/vendor/github.com/hashicorp/terraform-plugin-sdk/helper/schema/resource.go:455 +0x119
2020-02-13T11:42:59.515+0100 [DEBUG] plugin.terraform-provider-aws_v2.48.0_x4: github.com/hashicorp/terraform-plugin-sdk/internal/helper/plugin.(*GRPCProviderServer).ReadResource(0xc00000ef30, 0x6cf35a0, 0xc0008833b0, 0xc000aa9b80, 0xc00000ef30, 0xc0008833b0, 0xc000b3ea80)
2020-02-13T11:42:59.515+0100 [DEBUG] plugin.terraform-provider-aws_v2.48.0_x4:  /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-aws/vendor/github.com/hashicorp/terraform-plugin-sdk/internal/helper/plugin/grpc_provider.go:525 +0x3d8
2020-02-13T11:42:59.515+0100 [DEBUG] plugin.terraform-provider-aws_v2.48.0_x4: github.com/hashicorp/terraform-plugin-sdk/internal/tfplugin5._Provider_ReadResource_Handler(0x6227320, 0xc00000ef30, 0x6cf35a0, 0xc0008833b0, 0xc000af98c0, 0x0, 0x6cf35a0, 0xc0008833b0, 0xc0002cca10, 0x61)
2020-02-13T11:42:59.515+0100 [DEBUG] plugin.terraform-provider-aws_v2.48.0_x4:  /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-aws/vendor/github.com/hashicorp/terraform-plugin-sdk/internal/tfplugin5/tfplugin5.pb.go:3153 +0x217
2020-02-13T11:42:59.516+0100 [DEBUG] plugin.terraform-provider-aws_v2.48.0_x4: google.golang.org/grpc.(*Server).processUnaryRPC(0xc0003ea000, 0x6d13be0, 0xc0001f6a80, 0xc0008de400, 0xc000771ad0, 0xa01fdb0, 0x0, 0x0, 0x0)
2020-02-13T11:42:59.516+0100 [DEBUG] plugin.terraform-provider-aws_v2.48.0_x4:  /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-aws/vendor/google.golang.org/grpc/server.go:995 +0x460
2020-02-13T11:42:59.516+0100 [DEBUG] plugin.terraform-provider-aws_v2.48.0_x4: google.golang.org/grpc.(*Server).handleStream(0xc0003ea000, 0x6d13be0, 0xc0001f6a80, 0xc0008de400, 0x0)
2020-02-13T11:42:59.516+0100 [DEBUG] plugin.terraform-provider-aws_v2.48.0_x4:  /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-aws/vendor/google.golang.org/grpc/server.go:1275 +0xd97
2020-02-13T11:42:59.516+0100 [DEBUG] plugin.terraform-provider-aws_v2.48.0_x4: google.golang.org/grpc.(*Server).serveStreams.func1.1(0xc0001485f0, 0xc0003ea000, 0x6d13be0, 0xc0001f6a80, 0xc0008de400)
2020-02-13T11:42:59.516+0100 [DEBUG] plugin.terraform-provider-aws_v2.48.0_x4:  /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-aws/vendor/google.golang.org/grpc/server.go:710 +0xbb
2020-02-13T11:42:59.516+0100 [DEBUG] plugin.terraform-provider-aws_v2.48.0_x4: created by google.golang.org/grpc.(*Server).serveStreams.func1
2020-02-13T11:42:59.516+0100 [DEBUG] plugin.terraform-provider-aws_v2.48.0_x4:  /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-aws/vendor/google.golang.org/grpc/server.go:708 +0xa1
2020-02-13T11:42:59.518+0100 [DEBUG] plugin: plugin process exited: path=/Users/sjourdan/.terraform.d/plugins/darwin_amd64/terraform-provider-aws_v2.48.0_x4 pid=62251 error="exit status 2"
2020/02/13 11:42:59 rpc error: code = Canceled desc = context canceled
[...]
2020/02/13 11:43:01 Refreshing state... aws_kms_alias.tfer--alias-002F-cs-002D-kms-002D-1
2020/02/13 11:43:01 rpc error: code = Canceled desc = context canceled
2020/02/13 11:43:01 ERROR: Unable to refresh resource tfer--alias-002F-aws-002F-dynamodb
2020/02/13 11:43:01 ERROR: Unable to refresh resource tfer--alias-002F-aws-002F-ebs
2020/02/13 11:43:01 ERROR: Unable to refresh resource tfer--alias-002F-aws-002F-elasticfilesystem
2020/02/13 11:43:01 ERROR: Unable to refresh resource tfer--alias-002F-aws-002F-es
2020/02/13 11:43:01 ERROR: Unable to refresh resource tfer--alias-002F-aws-002F-glue
2020/02/13 11:43:01 ERROR: Unable to refresh resource tfer--alias-002F-aws-002F-kinesisvideo
2020/02/13 11:43:01 ERROR: Unable to refresh resource tfer--alias-002F-aws-002F-rds
2020/02/13 11:43:01 ERROR: Unable to refresh resource tfer--alias-002F-aws-002F-redshift
2020/02/13 11:43:01 ERROR: Unable to refresh resource tfer--alias-002F-aws-002F-s3
2020/02/13 11:43:01 ERROR: Unable to refresh resource tfer--alias-002F-aws-002F-ssm
2020/02/13 11:43:01 ERROR: Unable to refresh resource tfer--alias-002F-aws-002F-xray
2020/02/13 11:43:01 ERROR: Unable to refresh resource tfer--alias-002F-cs-002D-kms-002D-1
2020-02-13T11:43:01.750+0100 [DEBUG] plugin: plugin exited
2020/02/13 11:43:01 aws Connecting....
2020/02/13 11:43:01 aws save kms
2020/02/13 11:43:01 aws save tfstate for kms
```

### Steps To Reproduce

- Use this simple KMS terraform: 

```
resource "aws_kms_key" "kms_key_1" {
  description = "KMS Key 1"
  key_usage   = "ENCRYPT_DECRYPT"
  tags = {
    Name      = "KMS Key 1"
  }
}

resource "aws_kms_alias" "kms_key_alias_1" {
  name          = "alias/cs-kms-1"
  target_key_id = aws_kms_key.kms_key_1.key_id
}
```

- execute terraformer for import both resources (key & alias): 

```
$ ./terraformer-all-darwin-amd64-0.8.6 import aws --regions=eu-west-3 --resources kms
```

### Additional Context

- a

### References

- https://github.com/terraform-providers/terraform-provider-aws/issues/10280

**Comment by meshuga** [comment]
Those aliases should not be detected by terraformer in first place. They are used only internally by AWS services and users don’t much have use of it.

**Comment by sjourdan** [comment]
You're right! yet the following resource `aws_kms_alias` (`kms_key_alias_1 `) should be detected as it's user created:

```
2020/02/13 11:43:01 ERROR: Unable to refresh resource tfer--alias-002F-cs-002D-kms-002D-1
```

**Comment by ksunil6266** [comment]
Any update here, it is very pain full to handle it, Customer managed keys should be detected.
